index.ts

```diff
@@ -0,0 +1,110 @@
+import { launch } from './run.js'
+import {
+  DynamoDBClient,
+  PutItemCommand,
+  PutItemCommandOutput,
+  UpdateTableCommand,
+} from '@aws-sdk/client-dynamodb'
+import _arcFunctions from '@architect/functions'
+import { readdirSync, readFileSync } from 'node:fs'
+import path from 'node:path'
+import { marshall } from '@aws-sdk/util-dynamodb'
+
+let local: Awaited<ReturnType<typeof launch>>
+
+export const deploy = {
+  async services() {
+    if (process.env.ARC_DB_EXTERNAL) local = await launch()
+  },
+}
+
+export const sandbox = {
+  // @ts-expect-error: The Architect plugins API has no type definitions.
+  async start({ inventory: { inv }, arc }) {
+    const dynamodbClient = new DynamoDBClient({
+      region: inv.aws.region,
+      endpoint: `http://localhost:${process.env.ARC_TABLES_PORT}`,
+      requestHandler: {
+        requestTimeout: 3_000,
+        httpsAgent: { maxSockets: 500 }, // Increased from default to allow for higher throughput
+      },
+    })
+    const seedFile = arc['architect-plugin-dynamodb-local'].find(
+      (item: string[]) => item[0] == 'seedFile'
+    )[1]
+    const client = await _arcFunctions.tables()
+    if (seedFile) {
+      if (['sandbox-seed.json', 'sandbox-seed.js'].includes(seedFile)) {
+        console.log(
```

Comment:
This log message is a bit cryptic. I didn't understand it at all until I read your README. Please rewrite it.

Reply:
To what? I don't think it should be too long

Reply:
I don't know, but it was a bit opaque. The wording in the README was clear enough; maybe start from the text you have in the README and try to abridge it.